tests/test_utils.py

```diff
@@ -65,6 +66,24 @@ def test_make_atari_env(env_id, n_envs, wrapper_kwargs):
         assert np.max(np.abs(reward)) < 1.0
 
 
+def test_vec_env_kwargs(tmp_path):
+    monitor_dir = tmp_path / "test_make_vec_env/"
+    env = make_vec_env(
+        "MountainCarContinuous-v0",
+        n_envs=1,
+        monitor_dir=monitor_dir,
+        seed=0,
+        vec_env_cls=SubprocVecEnv,
```

Comment:
why not the default DummyVecEnv? and why with a monitor wrapper?

Reply:
Oh I merely copied over from the next test function, but it's not necessary. I'll remove lines 74 and 76.